server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/DeleteAction.java

```diff
@@ -57,7 +57,7 @@ public void define(NewController controller) {
       .setPost(true)
       .setHandler(this);
 
-    qProfileWsSupport.createOrganizationParam(action)
+    QProfileWsSupport.createOrganizationParam(action)
```

Comment:
could/should be merged with previous commit (same title and same change, just at a second location)